### PR TITLE
init run configuration

### PR DESCRIPTION
**Pull request by jdneo**

### CODE REVIEW

File: PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/intellij/actions/MavenWebDeployAction.java [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+        }
+
+        ApplicationManager.getApplication().invokeLater(() -> {
+            runConfiguration(project);
+        });
+    }
+
+    private void runConfiguration(Project project) {
+        final RunManagerImpl manager = RunManagerImpl.getInstanceImpl(project);
+        RunnerAndConfigurationSettings settings = manager.findConfigurationByName(configType.getDisplayName());
```
Comment (adashen) [inline PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/intellij/actions/MavenWebDeployAction.java]: use a param to specify the default configuration name.

File: PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/intellij/actions/MavenWebDeployAction.java [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+        boolean shouldCreateBeforeRunTask = (settings == null);
+        List<BeforeRunTask> beforeRunTasks = null;
+        if (shouldCreateBeforeRunTask) {
+            final ConfigurationFactory factory =
+                    configType != null ? configType.getConfigurationFactories()[0] : null;
+            settings = manager.createConfiguration(configType.getDisplayName(), factory);
```
Comment (adashen) [inline PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/intellij/actions/MavenWebDeployAction.java]: same as above. how about use project name as default name?

**Comment by jdneo** [comment]
retest this please

**Comment by jdneo** [comment]
retest this please

File: PluginsAndFeatures/azure-toolkit-for-intellij/resources/META-INF/plugin.xml [hunk 3]
```diff
@@ -62,12 +62,15 @@
...
 
+    <configurationType implementation="com.microsoft.intellij.runner.webapp.WebAppConfigurationType"/>
     <configurationType implementation="com.microsoft.azure.hdinsight.spark.run.configuration.RemoteDebugRunConfigurationType"/>
+    <programRunner implementation="com.microsoft.intellij.runner.webapp.webappconfig.WebAppRunner"/>
     <programRunner implementation="com.microsoft.azure.hdinsight.spark.run.SparkBatchJobDebuggerRunner" />
     <programRunner implementation="com.microsoft.azure.hdinsight.spark.run.SparkBatchJobRunner" />
-
   </extensions>
 
+
```
Comment (adashen) [inline PluginsAndFeatures/azure-toolkit-for-intellij/resources/META-INF/plugin.xml]: remove unneeded blank line

**Pull request merged**